src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py

```diff
@@ -92,6 +148,33 @@ def setup(self, parameters):
 
         return
 
+    def algorithm(self, parameters):
+        """
+        _algorithm_
+
+        Poll the database for uninjected files and attempt to inject them into
+        PhEDEx.
+        """
+        logging.info("Running PhEDEx injector poller algorithm...")
+
+        self.pollCounter += 1
+
+        if self.blocksToRecover:
+            logging.info("""PhEDExInjector Recovery:
```

Comment:
We better make it a warning logging and rephrase the message, "we" actually don't need to check it, but the agent will.